src/aspire/abinitio/sync_voting.py

```diff
@@ -13,6 +13,38 @@ class SyncVotingMixin(object):
     which are shared by CLSynVoting and CLSymmetryC3C4
```

Comment:
Sync, and string updates, but we can wait to cleanup the doc until the code settles. Some questions here.